person.rb

```diff
@@ -13,3 +13,24 @@ def correct_name
     name
   end
 end
+
+class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] According to the project requirement, Getters and Setters for @rentals should be present. I recommend you `attr_accessor :name, :age, :rentals` format for all the variables 👍🏽

Reply:
Thank you for the awesome review, @AmaduKamara . I'll start working on the changes right away!